app/policies/log_policy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class LogPolicy < TeamDependantPolicy
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    @user.is_a?(User::Human) && team.teammember?(@user.id)
+  end
+
+  def show?
+    team.teammember?(@user.id)
+  end
+
+  protected
+
+  def team
+    @record
```

Comment:
what represents record here?
shouldn't this somehow point from the log entry to the team through associated encryptable ?

Reply:
`@record` is a `Team` object. The way I understand it is that `Team` already represents the `Team` where the `encryptable` is in.

Reply:
@record should be either a PaperTrail::Version entry or a list of those.
so tell the policy how to reach team here. usally the team associated to the encryptable